### PR TITLE
Publicar Introdução à estilometria com Python

### PR DESCRIPTION
**Pull request by DanielAlvesLABDH**
*Publicar a tradução da lição Introduction to stylometry with Python*

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by DanielAlvesLABDH** [comment]
Hi @anisa-hawes, this PR fails because of a broken link in the lesson https://programminghistorian.org/en/lessons/extracting-illustrated-pages. The broken link is https://requests.readthedocs.io/en/master/. The main website works (https://requests.readthedocs.io/en/latest/), but I can't identify the correct or updated link. Can you help? Thanks

**Comment by anisa-hawes** [comment]
Hello @DanielAlvesLABDH,

In this lesson, I updated the link to José Calvo Tello’s article, ‘Entendiendo Delta desde las Humanidades’ in _Caracteres_ journal because <http://revistacaracteres.net/revista/vol5n1mayo2016/entendiendo-delta/> hit a 503 Error. I found the alternative <https://www.academia.edu/25792923/Caracteres_Vol_5_N_1> and [perma.cc](http://perma.cc/) allows us to capture the whole Issue, which you can view here: <https://perma.cc/LNF3-QP8V>.

A.

**Comment by DanielAlvesLABDH** [comment]
Dear @anisa-hawes, thank you again for your work with this. Since both PR have passed the checks I can ask for a final review from the Portuguese team and then publish, correct?

**Comment by anisa-hawes** [comment]
Yes!

File: pt/licoes/introducao-estilometria-python.md [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+
+# O *corpus* - Contextualização
+
+No [exemplo original deste tutorial em inglês](/en/lessons/introduction-to-stylometry-with-python), utilizaram-se os [papéis federalistas](https://perma.cc/DW5V-MH5W) como um exemplo de aplicação de estilometria, utilizando as técnicas que serão apresentadas para inferir a autoria dos textos contestados dentro do conjunto de documentos que configura o *corpus*.[^7]
+Como na língua portuguesa não temos um conjunto de textos que possua estas mesmas características, no exemplo que apresentaremos traremos um total de 15 obras completas de 5 autores diferentes, três deles portugueses e dois brasileiros, todos romancistas do século XIX, disponibilizadas pelo [Projeto Gutenberg](https://perma.cc/5PRR-TM3D). Utilizaremos duas obras de cada autor para definir seus respectivos estilos e uma terceira para constituir o conjunto de testes, para avaliarmos se as técnicas utilizadas realizarão a inferência correta de autoria através do grau de similariade de cada obra deste conjunto com o estilo obtido de cada autor.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-estilometria-python.md]: Corrigir `similariade` por  `similaridade`

File: pt/licoes/introducao-estilometria-python.md [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+
+Os autores e obras utilizadas são os seguintes:
+
+| Autor | Obra 1 | Obra 2 | Obra 3 |
+| --------- | --------- | --------- | --------- |
+| [Machado de **Assis**](https://perma.cc/6BMU-UKZL) (Brasil)| [Quincas Borba](https://www.gutenberg.org/ebooks/55682) (**55682**) | [Memorias Postumas de Braz Cubas](https://www.gutenberg.org/ebooks/54829) (**54829**) | [Dom Casmurro](https://www.gutenberg.org/ebooks/55752) (**55752**) |
```
Comment (ericbrasiln) [inline pt/licoes/introducao-estilometria-python.md]: Aqui se se pretende manter a grafia como no documento do link é preciso substituir `Postumas` por `Posthumas`. Se não, é melhor colocar a grafia atual, com os acentos. Acho que isso vale para todos os títulos de livros.

**Review by ericbrasiln — COMMENTED** [review]
@DanielAlvesLABDH 
Encontrei 2 pequenos erros e comentei nas linhas respectivas.

**Review by ericbrasiln — APPROVED** [review]
Aprovado. 🚀

**Pull request merged**